src/TodoButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function TodoButton({ todo, onClickRemove }) {
+  return (
+    <button type="button" onClick={() => onClickRemove(todo)}>완료</button>
```

Comment:
```suggestion
    <button
       type="button"
       onClick={() => onClickRemove(todo)}
     >
       완료
     </button>
```

`props`가 많은 경우 이렇게 줄바꿈 해줬을 때 코드 가독성을 비교하면 어떤게 더 나으신가요?

Reply:
props가 2개정도면 괜찮다고 생각하는데 3개 이상이면 줄바꿈해주는 게 좋은 것 같습니다.
간혹 컴포넌트 라인이 너무 길어져서 위아래로 움직이는 게 가독성을 해치기도 하더라구요.
기환님 생각은 어떠신가요?

Reply:
저는 2개 이상부터는 줄바꿈을 해주는 편인데 스타일은 개인의 의견에 따라 다를 수 있어서 참고정도만 해주시면 될 것 같아요 ;)

그냥 HTML 태그니 atttribute 라고 불러주는 게 맞겠네요 ;)

저는  보통  컴포넌트락고 바라보고 속성들을  넘겨준다고 생각하고 부르다보니 `props` 라고 불렀네요.
React 문서에 `props` 의  정의를 살펴보면 다음과 같이 나와있어요.

> When React sees an element representing a user-defined component, it passes JSX attributes and children to this component as a single object. We call this object “props”.

참고:
- [components-and-props](https://reactjs.org/docs/components-and-props.html)

Reply:
기환님 생각을 들어볼 수 있어서 좋네요 ㅎㅎ

그리고 React에서 사용하는 React.element이니 여러 속성을 통칭해서 말할 때
Props 라고 해도 될거 같기도 한데요? ㅎㅎ (공식문서 감사합니닷)

Reply:
말씀하신 부분도 맞는 것 같아요!
보통은 `props` 라고 많이 부르기 때문에 이해하기도 더 쉽지 않을까 생각이드네요.